docs: Improve Quick start instructions

waldyrious opened the pull request:
<!-- Please use English language -->
<!-- Please don't delete this template -->

<!-- PULL REQUEST TEMPLATE -->
<!-- (Update "[ ]" to "[x]" to check a box) -->

**Summary**

Re-submission of #1458, this time targeting the correct branch :)

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Repo settings
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

Koooooo-7 reviewed (approved):
thx.

trusktr reviewed (changes requested):
I think that last part maybe should be kept, and be "See the `el` option"?

trusktr commented:
About the snippet mentioning python, we should probably replace that with instructions on docsify CLI, or just use the tools we already have for managing Docsify, like Node/npm and `npx serve .`.

Otherwise we're mentioning a tool that isn't related to Docsify, and we're not documenting that. We're all using Node/npm to develop and consume Docsify.

waldyrious commented:
> I think that last part maybe should be kept, and be "See the `el` option"?

I added that link right above the code sample, which is why I removed that line. Adding it immediately below again would be redundant, and potentially even confusing IMO (I would have assumed it to be a typo or overlook of some kind).

If you prefer it at the bottom, then I'd suggest not linking "configure" in the mention above the code sample, which would justify the line below with the link. What do you think?

sy-records commented:
> About the snippet mentioning python, we should probably replace that with instructions on docsify CLI or just use the tools we already have for managing Docsify, like Node/npm and `npx serve .`. But this can be a separate PR.
> 
> Otherwise we're mentioning a tool that isn't related to Docsify, and we're not documenting that. We're all using Node/npm to develop and consume Docsify.

docsify-cli is included in the previous content, python is another way.

anikethsaha commented:
@trusktr you have some requested change. Is it good to review or still needs to be addressed ?

waldyrious commented:
> @trusktr you have some requested change. Is it good to review or still needs to be addressed ?

I responded to his comment with some considerations and proposed alternatives. I am happy to make changes once there's agreement on how to proceed.
